### PR TITLE
Benchmark of GitBucket / Gogs / Gitea / GitLab on Raspberry Pi 3

### PR DESCRIPTION
**Pull request by takezoe**
The original article by @kounoike at http://qiita.com/kounoike/items/7c42db90c42e280e1cab

### CODE REVIEW

**Comment by takezoe** [comment]
@kounoike Translated your article. Could you review this pull request?

Images are linked to Japanese version for now, but I would like to replace them before merging this pull request.

**Comment by kounoike** [comment]
Thanks for translation! I sent google spreadsheet's URL by twitter DM.

and I forgot GitLab's `free -m` result. Please add follows:

```
pi@raspberrypi:~ $ free -m
             total       used       free     shared    buffers     cached
Mem:           925        820        104         26         27        214
-/+ buffers/cache:        579        346
Swap:           99          0         99
```

File: _posts/2017-03-27-benchmark-of-gitbucket.markdown [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+![Results of top page after logged-in](https://qiita-image-store.s3.amazonaws.com/0/53147/e33d4c2e-661d-e8f2-fac3-44044bd7d64f.png)
+
+GitLab is slow a little, but others looks roughly same.
+
+### User page by number of repositories
+
+Created n repositories (n = 1, 10, 50, 100) by the test account and measured time to display the user page.
+
+Note that GitBucket displays all repositories at once, Gogs / Gitea paginates them.
```
Comment (kounoike) [inline _posts/2017-03-27-benchmark-of-gitbucket.markdown]: GitLab also paginates them.(Sorry I add it to Qiita now)

File: _posts/2017-03-27-benchmark-of-gitbucket.markdown [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+![clone-http](https://qiita-image-store.s3.amazonaws.com/0/53147/8f4b11ee-4cc3-e8c5-26e9-8935bc1bbcb3.png)
+
+![push-ssh](https://qiita-image-store.s3.amazonaws.com/0/53147/602e9010-6d8d-2700-35ef-d0d0a2b9a3af.png)
+
+![clone-ssh](https://qiita-image-store.s3.amazonaws.com/0/53147/256d24d3-d296-8772-fea0-7656a2a25246.png)
+
+In pushing via HTTP, GitLab is much slow, others are side by side. In cloning via HTTP, all products are side by side. In pushing / cloning via SSH, GitLab is slow. Gogs looks better in others, but it might be an error of measurement.
+
+I got errors in pushing to Gitea, but it looked to be working fine.
```
Comment (kounoike) [inline _posts/2017-03-27-benchmark-of-gitbucket.markdown]: maybe, s/errors/error messages/ ?

**Pull request merged**